### PR TITLE
Add IPNI org profile

### PR DESCRIPTION
**Pull request by masih**
Add the initial GitHub org profile for IPNI with links to resources that are most useful to implementers, contributors and the community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # .github
-:octocat: GitHub Org Readme
+:octocat: IPNI GitHub Organisation README
```
Comment (gammazero) [inline README.md]:
```suggestion
:octocat: IPNI GitHub Organization README
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+![banner](banner.png)
+
+# IPNI
+Welcome to the InterPlanetary Network Indexer (IPNI) organization on GitHub.
+
+IPNI is a content routing system optimised for large scale providers that facilitates bulk ingestion of millions of CIDs, and provides a super fast lookup over a simple HTTP REST API.
```
Comment (gammazero) [inline profile/README.md]:
```suggestion
IPNI is a content routing system optimized to take billions of CIDs from large-scale data providers, and allow fast lookup of provider information using these CIDs over a simple HTTP REST API.
```

**Review by gammazero — APPROVED** [review]
See suggestions

**Comment by masih** [comment]
Thank you @gammazero 👍

**Pull request merged**